### ISSUE TITLE
If trying to move both hands, can't move right hand

### ISSUE THREAD

**Issue by liam-mulhall** [phetsims/ratio-and-proportion#343]
### Test Device

MacBook Air

### Operating System

macOS 10.15.7

### Browser

Safari 14.0.2

### Description

For https://github.com/phetsims/QA/issues/601. If using Voice Over on macOS, and you navigate to "Both Hands", you can move the left hand using W and S, but you can't move the right hand using the up and down arrow keys. If you use the up and down arrow keys, you'll be interacting with Voice Over instead of the sim.

I'm fairly certain I use default Voice Over settings, so please let me know if there is something I need to change in my settings.

<details>
<summary><b>Troubleshooting Information</b></summary>

Name: ‪Ratio and Proportion‬
URL: https://phet-dev.colorado.edu/html/ratio-and-proportion/1.0.0-rc.1/phet/ratio-and-proportion_all_phet.html
Version: 1.0.0-rc.1 2021-01-26 23:41:16 UTC
Features missing: touch
User Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0.2 Safari/605.1.15
Language: en
Window: 1440x837
Pixel Ratio: 1/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0 (1.0)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 15 uniform: 1024
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 8192x8192
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by terracoda** [comment]
I don't think I have experienced this issue.

**Comment by zepumph** [comment]
@liam-mulhall, let's pair on this sometime later this week. I would like to witness the behavior more intimately. Before that though, can you please test a couple sims on master that use the grab drag interaction, for example Friction or BASE to see if you are able to move the book/balloon with w/s but not with arrow keys? That is the same application role elelment, though you have to use a button to enter that interaction.

**Comment by liam-mulhall** [comment]
Sounds good, @zepumph.

**Comment by liam-mulhall** [comment]
No longer seeing this on 11.2.

**Comment by liam-mulhall** [comment]
@zepumph and I are leaning toward this being a "sighting," to use JB terminology. I will take a look at this again when I have 10.15 back on my laptop.

**Comment by zepumph** [comment]
Sounds good. I will count this as non-blocking for now, but we can always promote if the "sighting" because more consistently reproducible.

**Comment by zepumph** [comment]
@liam-mulhall I'm going to close this. Please reopen if you encounter it again.
